### ISSUE TITLE
Add autocomplete fields to registration and login forms.

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#5185]
A newer requirement of WCAG 2.1 is to use autocomplete on all non-sensitive form fields. For example:

```
<input type="text" name="name" autocomplete="given-name">
```

A full list of valid values are https://www.w3.org/TR/WCAG21/#input-purposes.

When an appropriate autocomplete type exists, it should be added to the registration and login forms.

**Comment by thinkbulecount2** [comment]
PR: https://github.com/pkp/pkp-lib/pull/5986

cc: @NateWr
